.rubocop_todo.yml

```diff
@@ -1,12 +1,12 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 10000`
-# on 2021-01-29 08:53:40 UTC using RuboCop version 1.8.1.
+# on 2021-03-03 09:15:46 UTC using RuboCop version 1.10.0.
 # The point is for the user to remove these configuration records
```

Comment:
Lots of files removed/significantly changed so regenerated to remove unneccessary ignores